flytekit/extend/backend/base_agent.py

```diff
@@ -319,8 +320,11 @@ async def _create(
         self: PythonTask, task_template: TaskTemplate, output_prefix: str, inputs: Dict[str, Any] = None
     ) -> ResourceMeta:
         ctx = FlyteContext.current_context()
+        es = ctx.new_execution_state().with_params(mode=ExecutionState.Mode.TASK_EXECUTION)
```

Comment:
Change the execution mode to `TASK_EXECUTION`, so that flytekit will updalod the flytefile to remote during the serialization.

Reply:
since this [PR](https://github.com/flyteorg/flytekit/commit/bd4c7085e8cc6068900144bb55548e7c6c56f7c3), flytefile will not be uploaded in the `LOCAL_TASK_EXECUTION` mode